src/main/routes/enter-claim-number.ts

```diff
@@ -0,0 +1,16 @@
+import * as express from 'express'
+import { ErrorHandling } from 'common/errorHandling'
+import { Paths } from 'app/paths'
+/* tslint:disable:no-default-export */
+
+export default express.Router()
+  .get(Paths.enterClaimNumberPage.uri,
+    ErrorHandling.apply(async (req: express.Request, res: express.Response): Promise<void> => {
+      res.render('not-implemented-yet')
```

Comment:
We could redirect to `/eligibility` now which is not implemented yet but @jacmurphy is working on it. Otherwise please let @jacmurphy know he has to update this as a part of his story.